WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.java

```diff
@@ -254,7 +257,7 @@ public void onClick(View v) {
             }
         });
 
-        rootView.findViewById(R.id.row_logout).setOnClickListener(new View.OnClickListener() {
+        rootView.findViewById(R.id.row_wpcom_connect).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 if (mAccountStore.hasAccessToken()) {
```

Comment:
^ ignore this for now, we have another ticket for this problem https://github.com/wordpress-mobile/WordPress-Android/issues/4573